cookbook/integrations/flytekit_plugins/greatexpectations/README.rst

```diff
@@ -0,0 +1,111 @@
+.. great_expectations::
+
+Great Expectations
+==================
+
+**Great Expectations** is a Python-based open-source library for validating, documenting, and profiling your data. 
+It helps to maintain data quality and improve communication about data between teams.
+
+The goodness of data validation in Great Expectations can be integrated with Flyte to validate the data moving in and out of 
+the pipeline entities you may have defined in Flyte. This helps establish stricter boundaries around your data to 
+ensure that everything's as you expected and data will not crash your pipelines anymore unexpectedly!
+
+How to Define Your Integration
+------------------------------
+
+Great Expectations supports native execution of expectations against various :ref:`Datasources <how_to_guides__configuring_datasources>`, 
+such as Pandas dataframes, Spark dataframes, and SQL databases via SQLAlchemy.
+
+We're supporting two Flyte types that should suit Great Expectations' ``Datasources``:
+
+- :py:class:`flytekit.types.file.FlyteFile`: FlyteFile supports remote (and, of course, local) datasets. If a remote URI is given, 
+  the plugin downloads data to a user-given file which Great Expectations later validates.
+- :py:class:`flytekit.types.schema.FlyteSchema`: FlyteSchema supports tabular data, which the plugin will convert into a parquet file 
+  and validate the data using Great Expectations.
+
+.. note::
+  Flyte types are added because, in Great Expectations, we have the privilege to give a non-string (Pandas/Spark DataFrame) when using a 
+  :py:class:`RuntimeDataConnector <greatexpectations:great_expectations.datasource.data_connector.runtime_data_connector.RuntimeDataConnector>` 
+  but not when using an 
+  :py:class:`InferredAssetFilesystemDataConnector <great_expectations.datasource.data_connector.inferred_asset_filesystem_data_connector.InferredAssetFilesystemDataConnector>` 
+  or a 
+  :py:class:`ConfiguredAssetFilesystemDataConnector <great_expectations.datasource.data_connector.configured_asset_filesystem_data_connector.ConfiguredAssetFilesystemDataConnector>`. 
```

Comment:
We have to update links here.

Reply:
https://legacy.docs.greatexpectations.io/en/latest/autoapi/great_expectations/datasource/data_connector/configured_asset_filesystem_data_connector/index.html#great_expectations.datasource.data_connector.configured_asset_filesystem_data_connector.ConfiguredAssetFilesystemDataConnector